src/workflows-app/components/ImportGithub.js

```diff
@@ -22,7 +22,7 @@ const constraints = {
   },
 };
 
-const ImportGithub = ({ setLoading, signal, workspace, name, namespace, setSelectedSubHeader }) => {
+const ImportGithub = ({ setLoading, signal, workspace, workspace: { canCompute }, name, namespace, setSelectedSubHeader }) => {
```

Comment:
TOL: Would it be better to disable the sub-header itself when user doesn't have compute permissions rather than disabling individual components in here? And the tooltip can be also added to sub-header itself

Reply:
Let me ask Sara/Katie on this, and dockstore. Also, the ImportWorkflow page that Dockstore redirects to already prevents users from importing to workspaces with read access but maybe we should stop them even earlier and they can only navigate to Dockstore from workspaces that they have write or above access...